src/Illuminate/Support/Testing/Fakes/NotificationFake.php

```diff
@@ -174,7 +174,9 @@ public function sendNow($notifiables, $notification)
         }
 
         foreach ($notifiables as $notifiable) {
-            $notification->id = Uuid::uuid4()->toString();
+            if (! $notification->id) {
```

Comment:
This might sound a bit nitpicky, but wouldn't checking for `null` be preferable here to allow `'0'` to be used as an ID?

Reply:
Well a UUID would never be `0`, so not sure that makes sense.